# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2023/07
# default paths for models, schedules, and examples in site-packages/rfdiffusion
diff -ru RFdiffusion-1.1.0/rfdiffusion/inference/model_runners.py RFdiffusion-1.1.0_data_paths/rfdiffusion/inference/model_runners.py
--- RFdiffusion-1.1.0/rfdiffusion/inference/model_runners.py	2023-04-03 23:33:05.000000000 +0200
+++ RFdiffusion-1.1.0_data_paths/rfdiffusion/inference/model_runners.py	2023-07-11 12:51:31.790793525 +0200
@@ -14,7 +14,7 @@
 from rfdiffusion import util
 from hydra.core.hydra_config import HydraConfig
 import os
-
+from rfdiffusion import __path__ as rfdiffusion_dir 
 from rfdiffusion.model_input_logger import pickle_function_call
 import sys
 
@@ -63,7 +63,7 @@
         if conf.inference.model_directory_path is not None:
             model_directory = conf.inference.model_directory_path
         else:
-            model_directory = f"{SCRIPT_DIR}/../../models"
+            model_directory = f"{rfdiffusion_dir[0]}/models"
 
         print(f"Reading models from {model_directory}")
 
@@ -122,7 +122,7 @@
         if conf.inference.schedule_directory_path is not None:
             schedule_directory = conf.inference.schedule_directory_path
         else:
-            schedule_directory = f"{SCRIPT_DIR}/../../schedules"
+            schedule_directory = f"{rfdiffusion_dir[0]}/schedules"
 
         # Check for cache schedule
         if not os.path.exists(schedule_directory):
@@ -148,7 +148,7 @@
         if self.inf_conf.input_pdb is None:
             # set default pdb
             script_dir=os.path.dirname(os.path.realpath(__file__))
-            self.inf_conf.input_pdb=os.path.join(script_dir, '../../examples/input_pdbs/1qys.pdb')
+            self.inf_conf.input_pdb=os.path.join(rfdiffusion_dir[0], './examples/input_pdbs/1qys.pdb')
         self.target_feats = iu.process_target(self.inf_conf.input_pdb, parse_hetatom=True, center=False)
         self.chain_idx = None
 
